test/blip/components/logic/actions.test.js

```diff
@@ -60,6 +60,9 @@ describe('actions', function() {
             diagnosisDate: '2010-01-01',
           },
         },
+        settings: {
+          previousSetting: true,
```

Comment:
I think it is just testing that the new setting is merged with any existing settings.